### ISSUE TITLE
Tutorial on using the box-shadow support for themes

### ISSUE THREAD

**Issue by justintadlock** [WordPress/developer-blog-content#51]
### Discussed in https://github.com/WordPress/developer-blog-content/discussions/47

<div type='discussions-op-text'>

<sup>Originally posted by **justintadlock** January  3, 2023</sup>
Gutenberg 14.9 introduces a new "shadow" preset for theme.json: https://github.com/WordPress/gutenberg/pull/46813

This allows devs to create a set of custom box-shadow presets for use in their themes, which can be referenced in `theme.json` or templates.  We need a tutorial that covers how to implement this. 

Currently, we are still waiting on UI integration in the form of a component on the user's end.  Do y'all think it's worth waiting for the UI pieces to land before the tutorial?  

I'm up for writing this.</div>

**Comment by justintadlock** [comment]
Current progress (really rough): [index.md](https://github.com/WordPress/developer-blog-content/files/10363358/index.md)

I expect to have this post finished on Monday, Jan. 9.  It will be available as a Google Doc and ready for review.

Article images:

![shadow-sm-md-lg](https://user-images.githubusercontent.com/1816309/211106446-de1b45f7-26cc-4932-af37-dbc0f3ca2919.jpg)
![shadow-lg](https://user-images.githubusercontent.com/1816309/211106450-788bfcda-2fae-4e40-a0d6-f021e456f887.jpg)
![shadow-primary](https://user-images.githubusercontent.com/1816309/211106454-a267dde5-7033-40a1-8775-5cf6d40b2029.jpg)

**Comment by justintadlock** [comment]
Draft of tutorial is now on Google Docs and is ready for review: https://docs.google.com/document/d/1lfIt8PZuKDpvrrEHWwkd3CwKK9ksZreScOb_CsPyhZA/edit?usp=sharing

**Comment by mburridge** [comment]
Excellent article @justintadlock. Everything is very clearly explained. I think it's good to go. Great work.

**Comment by nalininsbs** [comment]
(Borrowed Nalini's laptop as mine is in repair. Did not notice, it linked me to her GitHub in error as she had been reviewing another item. Apologies to Nalini.)

**Comment by abhansnuk** [comment]
Hello @justintadlock , second review done. 

I have added comments to the google doc. Loved this article, easy for devs to follow. From your google doc, would a title alternative 'Using the box shadow feature for themes? 

Standard additions: Alt text, excerpt, categories for audience level (beginner, intermediate, advanced), props list. 

I have added a more general question in Slack about when we are calling an article a 'tutorial' and trying to see how this aligns with Learn WordPress.
